.github/workflows/rust.yml

```diff
@@ -1,10 +1,8 @@
 name: Rust + Webpack
 
 on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+  - push
+  - pull_request
```

Comment:
I changed this to be able to test but I see no reason you wouldn't want to run the tests on other branches, so I left it in. It will help contributors for sure.